sr_gui_change_controllers/src/sr_gui_change_controllers/change_controllers.py

```diff
@@ -260,14 +314,45 @@ def display_information(self, message):
                   "The output of the host side PID controller is sent to the motor " + \
                   "as a PWM demand. No effort controller is used for position control.\n" + \
                   "Teach Mode: No control is implemented on the host. The Effort " + \
-                  "demand is sent to the motor which implements it using a 5kHz control loop."
+                  "demand is sent to the motor which implements it using a 5kHz control loop.\n" + \
+                  "Direct PWM Commands: This is used for basic position control, " + \
+                  "and is used by default on a new hand. The PWM demand value is sent " + \
```

Comment:
what do you mean by "used by default on a new hand"?

Reply:
i found this in the palm_EDC_user_manual.pdf
![image](https://user-images.githubusercontent.com/45658238/149517148-2a0d3fa4-f230-412b-b81d-b4aad4999b90.png)

Reply:
I think that's referring to the types of command that can be sent to the motor, where PWM demand is a direct motor instruction (used with position control, trajectory control, direct PWM/effort), and Torque demand instead uses a Torque control loop on the motor board (used for teach mode only). So, PWM demand is default for a new hand, but that doesn't extend to 'Direct PWM Commands' in change_controllers.py

Reply:
do you think just removing that part sounds better:
`"Direct PWM Commands: This is used for basic position control. The PWM demand value is sent directly to the motor, unless there is a safety cutout.\n"`

Reply:
fixed [9932816](https://github.com/shadow-robot/sr_visualization_common/pull/35/commits/99328168fdda7403a372d20431ce791cec249269)